enacts/wat_bal/maproom_monit.py

```diff
@@ -44,6 +44,16 @@
 DR_PATH = f"{GLOBAL_CONFIG['daily']['zarr_path']}{DATA_PATH}"
 RR_MRG_ZARR = Path(DR_PATH)
 rr_mrg = calc.read_zarr_data(RR_MRG_ZARR)
+#rr_mrg["X"] = rr_mrg["X"].astype(np.single)
+#rr_mrg["Y"] = rr_mrg["Y"].astype(np.single)
+
+kaka, taw = xr.align(
+    rr_mrg,
+    xr.open_dataarray(Path(CONFIG["taw_file"])),
+    join="override",
+    exclude="T",
+)
```

Comment:
No point calling `align` here anymore, now that the only thing we use this copy of the data for is to calculate the max.

Reply:
I use it in the tile function see line 476. But then, if I put taw_max hard-coded in config file, and if file reading should rather be within callbacks, then I could put that reading and alignment in the tile function.

Reply:
one-to-last commit reorganizes according to our discussion.